cmd/kube-controller-manager/app/options/persistentvolumebindercontroller.go

```diff
@@ -63,6 +69,17 @@ func (o *PersistentVolumeBinderControllerOptions) Validate() []error {
 		return nil
 	}
 
+	cleanedList := strings.TrimSpace(o.DialHostCIDRDenylist)
```

Comment:
The validation logic should be consistent with `parseControllerDialhostIPDenylist`, so I don't think you should trim spaces here. Alternatively, you could refactor the parse function to take the string argument directly, and just call it here as well (check for the error, and through away the parsed list).

Reply:
Good point. Refactored to use netutils.ParseCIDRs which I think accomplishes your suggestion.

Reply:
Actually, with the "none" change suggested below I ended up doing exactly your refactoring.